2.15/2.15.7/index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<title>Title</title>
+</head>
+<body>
+### 2.15.7
+В самом начале выводится 2 пары сообщений:
+1) "Введите имя" и "Введите возраст"
+2) "Введите имя" и "Введите возраст"
+
+А потом вывести 2 алерта с инфой из введенных данных:
+"Этого человека зовут - Х. Он [еще молод], потому что ему Y лет."
+Причем фраза [еще молод] может поменяться на [уже взрослый], если введенный для него возраст был больше 18.
+<script>
+	function userInformation(name, age) {
+		let ageText = 'еще молод';
+
+		if (age > 18) {
+			ageText = 'уже взрослый';
+		}
+
+		return alert(`Этого человека зовут - ${name}. Он ${ageText}, потому что ему ${age} лет.`);
```

Comment:
you don't need `return` here because the function `getSum` shouldn't return anything, it should just show a message and that's all